ui/src/main/res/values/dimen.xml

```diff
@@ -27,12 +27,16 @@
     <dimen name="image_tile_vertical_width">98dp</dimen>
     <dimen name="image_tile_vertical_height">140dp</dimen>
 
+    <dimen name="fragment_detail_movies_top_spacing">100dp</dimen>
```

Comment:
Should I use percents for Backdrop image? And percents for block with poster and info?